### DISCUSSION TITLE
Restricted Mode with comment

### DISCUSSION THREAD

**Discussion by lotusdroid** [ReVanced/revanced-discussions#375] [Question]
Is it possible to create patches to make comment working in restricted mode?
Yeah. Youtube offer some p0rn content in short or feed(feed is less, usually no) but when turn on it, that video with "dirty" content is gone but i cannt see comment or comment to video
Thanks team very much

**Comment by TheJeterLP** [comment]
Not possible since we cannot filter videos.

**Reply by lotusdroid** [reply]
Thanks forr info

**Reply by oSumAtrIX** [reply]
The question was about making comments work in restricted mode, not filter videos in unrestricted mode. While the latter can also work because it is possible to filter videos via keywords (#41), showing the comments in restricted mode is also technically possible by simply disabling the restriction of displaying comments in the application.

**Reply by lotusdroid** [reply]
OMG. Thanks oSumAtrlX and ReVanced Team🥰

**Reply by TheJeterLP** [reply]
> The question was about making comments work in restricted mode, not filter videos in unrestricted mode. While the latter can also work because it is possible to filter videos via keywords (#41), showing the comments in restricted mode is also technically possible by simply disabling the restriction of displaying comments in the application.

Filtering videos with Keywords is possible. But you can't filter with content. So if the keyword doesn't match the video still goes through the filter.

**Reply by oSumAtrIX** [reply]
> Not possible since we cannot filter videos.

Hence filtering is possible by keywords (#41).

**Reply by TheJeterLP** [reply]
You can filter video titles and descriptions. But not video content

**Reply by oSumAtrIX** [reply]
That was never the question to begin with. He said "dirty" videos, which are required to be tagged on YouTube as +18 content which can be filtered to which you responded, that videos can not be filtered at all.

**Reply by TheJeterLP** [reply]
> That was never the question to begin with. He said "dirty" videos, which are required to be tagged on YouTube as +18 content which can be filtered to which you responded, that videos can not be filtered at all.

Well, what if they contain "dirty content" and are not tagged as 18+ and go through the naming filter?

**Reply by oSumAtrIX** [reply]
That's an issue on YouTube's end. Due to YouTube's regulations, that any adult video has to be tagged, we are allowed to say they are all tagged appropriately and filter them.
